lib/opensearch-aws-sigv4.rb

```diff
@@ -79,6 +79,8 @@ def verify_open_search
       def signature_url(path, params)
         host = @transport.transport.hosts.dig(0, :host)
         path = "/#{path}" unless path.start_with?('/')
+        params = params.clone
+        params.delete(:ignore)
```

Comment:
I'd also do `params.delete('ignore')` as well to account for this key as a string.
I'll add this in the version bump PR.